configs/todc-bootstrap-v3.json

```diff
@@ -1,6 +1,7 @@
 {
-  "index_name": "todc-bootstrap",
+  "index_name": "todc-bootstrap-v3",
```

Comment:
This attribute must be the same as the file name. Don't change it please

Reply:
Here's the text of an email conversation I was had:

My question:

> Is it possible for me to add/update indexes? This site is documentation for a Google styled theme for Bootstrap. The documentation includes the full Bootstrap documentation as well as documentation for any additional/modified components. Bootstrap includes documentation for multiple versions and I'd like to make the versions on the theme site searchable via their own indexes.

Response:

> Hey Tom,

> I would definitely recommend to add it into your configuration file as long as it is within the same website.

Does this mean that I should update my start_urls and selectors within the one configuration file?

```https://todc.github.io/todc-bootstrap/docs/3.3.7/,```
```https://todc.github.io/todc-bootstrap/docs/4.0/```

Thanks

Reply:
Don't worry, I noticed it was you.

As long as the new content is linked from the start url or its sub ones, it will be updated at the next crawl (every 24h)

Reply:
I'm looking for each version of the docs to be searched separately. I don't want to get hits back that point to the v3 docs when I'm in the v4 docs. Wouldn't this require separate indexes and configurations?

The urls given about are not the way the site is currently structured, but it's what I'm moving towards

Reply:
Hi,

If the different versions have the same DOM structure, you just have to add a new start url.

Otherwise you will have to use different selector key. I will support you in order to that.

Best,

Reply:
Hi,

Versions 3.x and 4.x would have different DOM structures. The selector keys in the current config are correct for v3.x. These are the keys for v4.x:

``` 
"selectors": {
    "lvl0": ".bd-toc-item.active > a",
    "lvl1": ".bd-content h1",
    "lvl2": ".bd-content h2",
    "lvl3": ".bd-content h3",
    "text": ".bd-content p, .bd-content li"
  },
```

Do I need to add these selectors to the existing ones?

```
  "selectors": {
    "lvl0": ".bs-docs-header h1, .bd-toc-item.active > a",
    "lvl1": ".bs-docs-container h1, .bd-content h1",
    "lvl2": ".bs-docs-container h2, .bd-content h2",
    "lvl3": ".bs-docs-container h3, .bd-content h3",
    "text": ".bs-docs-container p, .bs-docs-container li, .bd-content p, .bd-content li"
  },
```

Thanks

Reply:
It those selector have no conflict from one version to another and you don't have  any overlap, yes it will works since we do an OR between all of those.

Let me know when I should try it

Reply:
The selectors, currently, do not overlap between v3.x and v4.x. I've uploaded the v4 **content** to https://todc.github.io/todc-bootstrap/docs/4.0/

FYI - The updated CSS hasn't been uploaded and the links will probably not work.

Reply:
Thanks, please let us know as soon as your layout is ready and that your website is stable. We will crawl it once it is in production.

Reply:
Will do. Thanks

Reply:
I believe we're ready to go.

Thanks